io.catenax.material_flow_simulation_result/2.0.0/MaterialFlowSimulationResultAspect.ttl

```diff
@@ -0,0 +1,328 @@
+
+:MaterialFlowSimulationResultCharacteristic a samm:Characteristic ;
+   samm:preferredName "Material Flow Simulation Result Characteristic"@en ;
+   samm:description "Characteristic describing the aspect material flow simulation result"@en ;
+   samm:dataType :MaterialFlowSimulationResult .
+
+:MaterialFlowSimulationResult a samm:Entity ;
+   samm:preferredName "Material Flow Simulation Result"@en ;
+   samm:description "Result of a Material Flow Simulation"@en ;
+   samm:properties ( :description :runId :comment :expirationTimestamp :owner :dataQuality :shipments :timestamp ) .
+
+:description a samm:Property ;
+   samm:preferredName "Description"@en ;
+   samm:description "Verbal description"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "Please simulate asap" .
+
+:runId a samm:Property ;
+   samm:preferredName "RunId"@en ;
+   samm:description "Id of the simulation run"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "0fece48b-c8d1-4180-1a9caca6d67e" .
+
+:comment a samm:Property ;
+   samm:preferredName "Comment"@en ;
+   samm:description "Additional comments"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "successful simulation " .
+
+:expirationTimestamp a samm:Property ;
+   samm:preferredName "ExpirationTimestamp"@en ;
+   samm:description "Date and time when the simulation is expired"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-03-24T09:15:24.000Z"^^xsd:dateTime .
+
+:owner a samm:Property ;
+   samm:preferredName "Owner"@en ;
+   samm:description "Site identifier of the simulation originator"@en ;
+   samm:characteristic :OwnerCharacteristic .
+
+:dataQuality a samm:Property ;
+   samm:preferredName "Data Quality"@en ;
+   samm:description "Information on quality of simulation results"@en ;
+   samm:characteristic :DataQualityOptions ;
+   samm:exampleValue "0"^^xsd:positiveInteger .
+
+:shipments a samm:Property ;
+   samm:preferredName "Shipments"@en ;
+   samm:description "List of shipments"@en ;
+   samm:characteristic :ShipmentList .
+
+:timestamp a samm:Property ;
+   samm:preferredName "Timestamp"@en ;
+   samm:description "Timestamp of simulation run"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-03-09T14:13:42.806Z"^^xsd:dateTime .
+
+:Text a samm:Characteristic ;
+   samm:preferredName "Text"@en ;
+   samm:description "Describes a Property which contains plain text. This is intended exclusively for human readable strings, not for identifiers, measurement values, etc."@en ;
+   samm:dataType xsd:string .
+
+:OwnerCharacteristic a samm-c:SingleEntity ;
+   samm:preferredName "Owner Characteristic"@en ;
+   samm:description "Site identifier of the simulation originator"@en ;
+   samm:dataType :BusinessPartnerSite .
+
+:DataQualityOptions a samm-c:Enumeration ;
+   samm:preferredName "Data Quality Options"@en ;
+   samm:description "Possible values determining data quality"@en ;
+   samm:dataType xsd:positiveInteger ;
```

Comment:
please change here and at below line also data type to `nonNegativeInteger`

Reply:
has been changed to `nonNegativeInteger`